modules/ROOT/pages/configure-kubernetes.adoc

```diff
@@ -0,0 +1,19 @@
+= Configuring Kubernetes for Runtime Fabric
+
+To configure Kubernetes for Anypoint Runtime Fabric, review the following topics:
+
+* xref:rtf-k8s-practices.adoc[Kubernetes (K8s) Best Practices for Runtime Fabric] +
```

Comment:
Global: In all articles that show a list of articles like this one, consider using an ordered list instead of a bullet list if the user must read these topics in order.

Reply:
in this case, there is no order required. They can read whatever doc they want. Although, I'll apply your suggestion on those that they do.